resources/asciidoctor/spec/spec_helper.rb

```diff
@@ -28,16 +28,20 @@ def initialize warnings, result
   end
 end
 
+# Put asciidoctor into verbose mode so it'll log reference errors
+$VERBOSE = true
+
 ##
 # Convert an asciidoc string into docbook. If the conversion results in any
 # errors or warnings then raises a ConvertError.
 def convert input
   logger = Asciidoctor::MemoryLogger.new
   result = Asciidoctor.convert input, {
-      :safe => :unsafe,  # Used to include "funny" files.
-      :backend => :docbook45,
-      :logger => logger,
-      :doctype => :book,
+      :safe       => :unsafe,  # Used to include "funny" files.
+      :backend    => :docbook45,
+      :logger     => logger,
+      :doctype    => :book,
+      :verbose    => true,
```

Comment:
Should this use the `$VERBOSE` variable?

Reply:
Oh! Actually it doesn't do anything. Let me drop it.

Reply:
I tried this first because it feels like it should work, but it doesn't. Because :shrug: